grpc/gradle.properties

```diff
@@ -1,3 +1,3 @@
 kotlin.code.style=official
 
-release_version=2.0.0
\ No newline at end of file
+release_version=2.1.0
```

Comment:
We still have not created a release. Don't see any reason to increment version again

Reply:
grpc/gradle.properties is already updated to 2.1.0. I have updated to the same value

Reply:
Maybe we should rollback changes in grpc/gradle.properties then